gcp/databases/cloudsql/private_postgres/versions.tf

```diff
@@ -0,0 +1,29 @@
+terraform {
+  required_version = ">= 0.13"
```

Comment:
This is old. Has it been ran against the latest version, and are all attributes accounted for?

Reply:
@danperezsan, did this get confirmed?

Reply:
Successfully ran against latest terraform version